app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencySheet.kt

```diff
@@ -117,6 +117,7 @@ class MedicineFrequencySheet : BottomSheetActivity(), MedicineFrequencySheetUiAc
       BD -> R.id.medicineFrequencyBdRadioButton
       TDS -> R.id.medicineFrequencyTdsRadioButton
       QDS -> R.id.medicineFrequencyQdsRadioButton
+      is MedicineFrequency.Unknown -> throw AssertionError("Unknown medicine frequency $medicineFrequency")
```

Comment:
The reason we add this as an `Unknown` is so that we can have a graceful fallback in cases where an older version of the app may not know how to handle this new value. Throwing an error here defeats the purpose since it will cause an app crash.

We should just ignore `Unknown` values here instead of throwing an error.

Reply:
Yes, will do that.